src/components/language/dto/update-language.dto.ts

```diff
@@ -72,6 +72,9 @@ export abstract class UpdateLanguage {
 
   @NameField({ nullable: true })
   readonly leastOfTheseReason?: string;
+
+  @Field({ nullable: true })
+  readonly isSignLanguage?: boolean = false;
```

Comment:
```suggestion
  readonly isSignLanguage?: boolean;
```
We don't want it set back to false if it's omitted. Probably copy/paste error 😉

Reply:
🤦